torntools/scripts/content/travel/ttTravelAbroad.js

```diff
@@ -409,6 +409,14 @@ function addFilterToTable(list, title) {
                     <div class="tt-checkbox-wrap"><input type="checkbox" value="online">Online</div>
                     <div class="tt-checkbox-wrap"><input type="checkbox" value="idle">Idle</div>
                     <div class="tt-checkbox-wrap"><input type="checkbox" value="offline">Offline</div>
+                </div>
```

Comment:
Please provide proper formatting inside of HTML written in a JS file.

Reply:
Done.